design/spaces.go

```diff
@@ -57,6 +58,24 @@ var spaceSingle = JSONSingle(
 	space,
 	nil)
 
+// relationSpaces is the JSONAPI store for the spaces
+var relationSpaces = a.Type("RelationSpaces", func() {
+	a.Attribute("data", relationSpacesData)
+	a.Attribute("links", genericLinks)
+	a.Attribute("meta", a.HashOf(d.String, d.Any))
+})
+
+// relationSpacesData is the JSONAPI data object of the space relationship objects
+var relationSpacesData = a.Type("RelationSpacesData", func() {
+	a.Attribute("type", d.String, func() {
+		a.Enum("spaces")
+	})
+	a.Attribute("id", d.UUID, "UUID for the space", func() {
+		a.Example("6c5610be-30b2-4880-9fec-81e4f8e4fd76")
```

Comment:
@hectorj2f I don't really think we need an example for a UUID. This is probably generated automatically by goa.